### PR TITLE
📝 Update JWT auth documentation to use PyJWT instead of pyhon-jose

### PR DESCRIPTION
**Pull request by estebanx64**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5a6859263bdb4db1386b244fd7fe5710d0379223 at: https://604f2b88.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit f252dc622d44c8f8ad18a569fd0ca8b14e919fc4 at: https://d2e11754.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 6734f798a7e37665d0cdca135dfcc7391847e7eb at: https://ac0fd867.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 6734f798a7e37665d0cdca135dfcc7391847e7eb at: https://0c8011f7.fastapitiangolo.pages.dev

**Review by YuriiMotov — CHANGES_REQUESTED** [review]
Everything is perfect except one small mistake with code line highlighting.

**Review by yezz123 — APPROVED** [review]
As @YuriiMotov mentioned, everything is amazing, especially with the non-maintenance of python-jose. Therefore, pyjwt is the best alternative.

LGTM 🚀

File: docs/en/docs/advanced/security/oauth2-scopes.md [hunk 1]
```diff
@@ -406,7 +406,7 @@ In this exception, we include the scopes required (if any) as a string separated
     !!! tip
         Prefer to use the `Annotated` version if possible.
 
-    ```Python hl_lines="105  107-115"
+    ```Python hl_lines="105  108-116"
```
[the remark this answers is not in the document]
Comment (estebanx64) [reply docs/en/docs/advanced/security/oauth2-scopes.md:409]: You are right, thanks!

**Comment by tiangolo** [comment]
📝 Docs preview for commit 875c02576d84166f4d58dd17f99d455ea831512b at: https://83636c12.fastapitiangolo.pages.dev

**Review by YuriiMotov — APPROVED** [review]
Everything is good!
I paid special attention to highlighting the lines of code in all code blocks.
I also checked that all the necessary documentation pages have been corrected.

**Pull request merged**